make/dpcpp.inc

```diff
@@ -19,6 +19,7 @@ ifeq ($(os_name), linux)
     ifneq (, $(shell which dpcpp))
         override compiler:=dpcpp
     endif
+    CPLUS_FLAGS += -fopenmp-simd
```

Comment:
Shouldn't the `-fiopenmp-simd` be for dpcpp?

Reply:
Actally, I dont aware of "-fi" prefix.... just about "-f" for linux and "-q" for windows.

Reply:
As far as I know this option is for `icpx`: https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compiler-reference/compiler-options/compiler-option-details/offload-compilation-options-and-openmp-options-and-parallel-processing-options/fiopenmp-qiopenmp.html

Reply:
1) According to the comment https://jira.devtools.intel.com/browse/CMPLRLLVM-26270?focusedCommentId=12470803&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-12470803
-fopenmp-simd is workable for dpcpp...
 2) for icpx it works as well.